prac_05/wimbledon.py

```diff
@@ -0,0 +1,46 @@
+def main():
+    champions_data = read_file("wimbledon.csv")
+    champion_wins = count_champion_wins(champions_data)
+    countries = list_countries_of_champions(champions_data)
+    display_champion_wins(champion_wins)
+    display_countries_of_champions(countries)
+
+
+def read_file(filename):
+    champions_data = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        for line in in_file:
+            champions_data.append(line.strip().split(','))
+    return champions_data
+
+
+def count_champion_wins(champions_data):
+    score = {}
```

Comment:
It returns a dictionary where the keys are champion names and the values are the corresponding win counts